packages/rancher-istio/charts/templates/service-monitors.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.kiali.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: envoy-stats-monitor
+  namespace: istio-system
+  labels:
+    monitoring: istio-proxies
+spec:
+  selector:
+    matchExpressions:
+    - {key: istio-prometheus-ignore, operator: DoesNotExist}
```

Comment:
nit:
```suggestion
    - key: istio-prometheus-ignore
      operator: DoesNotExist
```

Reply:
this is an imported file so I'd prefer to leave as-is to simplify future imports